src/net/tls-impl.cc

```diff
@@ -0,0 +1,740 @@
+/*
+ * This file is open source software, licensed to you under the terms
+ * of the Apache License, Version 2.0 (the "License").  See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.  You may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+/*
+ * Copyright 2015 Cloudius Systems
```

Comment:
This header looks like a copy-and-paste. But in general, the copyright notice should reflect the original employer under whom this code was created, as work produced during employment typically belongs to the employer. If, and only *if*, there was a subsequent transfer of copyright to ScyllaDB (formerly known as Cloudius Systems), please update the header to use the current company name 'ScyllaDB' rather than 'Cloudius Systems'.